adapters/huaweiads/huaweiads.go

```diff
@@ -1396,3 +1431,245 @@ func getDigestAuthorization(huaweiAdsImpExt *openrtb_ext.ExtImpHuaweiAds, isTest
 		"response=" + computeHmacSha256(nonce+":POST:/ppsadx/getResult", apiKey) + "," +
 		"algorithm=HmacSHA256,usertype=1,keyid=" + huaweiAdsImpExt.KeyId
 }
+
+var MccList = map[int]string{
```

Comment:
Please consider moving this list to a separate file in this package

Reply:
The MCC list is a standard list, not Huawei specific, but Huawei is using the mcc to determine the origin country of the request, in order to decide which endpoint we should forward this request to. Therefore I recommend we keep the list in the code as it seems like Huawei is the only bidder that requires this set up.

Reply:
Sounds good. But can you please move those codes to a diff file in this package so as to not overload this file with so many lines of code and also just to logically separate the code for your adapter from the MCC list?